packages/admin/src/form/FinalFormRangeInput.tsx

```diff
@@ -74,21 +78,21 @@ const FinalFormRangeInputComponent: React.FunctionComponent<WithStyles<typeof st
                         <InputBase
                             name={`${name}.min`}
                             inputProps={{
-                                min: min,
-                                max: fieldValue.max,
-                                value: internalMinInput,
-                                type: "number",
+                                value: internalMinInput !== undefined ? internalMinInput : "",
+                                type: "text",
+                                placeholder: min,
                             }}
                             startAdornment={startAdornment}
                             endAdornment={endAdornment}
                             onBlur={() => {
-                                const minFieldValue = Math.min(internalMinInput, fieldValue.max);
-                                if (fieldValue.min !== minFieldValue) {
-                                    onChange({ ...fieldValue, min: minFieldValue < min ? min : minFieldValue });
-                                }
+                                const minFieldValue = Math.min(internalMinInput ? internalMinInput : min, fieldValue.max ? fieldValue.max : max);
+
+                                onChange({ ...fieldValue, min: minFieldValue < min ? min : minFieldValue });
```

Comment:
Now when I focus the field, do nothing and blur it you write max into it. That doesn't feel right.

And shouldn't (now that we support undefined value) "" input value set it back to undefined?

Reply:
ok i thought that would be ok, i will change that asap